src/App.js

```diff
@@ -1,15 +1,18 @@
 import { useEffect, useState } from "react";
 import './App.css';
+import Playlist from "./components/Playlist";
 import logo from './mlh-prep.png'
 
 function App() {
   const [error, setError] = useState(null);
   const [isLoaded, setIsLoaded] = useState(false);
   const [city, setCity] = useState("New York City")
   const [results, setResults] = useState(null);
+  const WEATHER_API_KEY = '3055aee78f153c35e0fa1c8d0afdaf90'
+  const [weather_condition, set_weather_condtion] = useState('')
```

Comment:
use camelCase for naming.
Lets keep consistency

Reply:
Okay